docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+
+You can open any number of directories from the command line by passing their paths to the `atom` command line tool. For example, you could run the command `atom ./hopes ./dreams` to open both the `hopes` and the `dreams` directories at the same time.
+
+When you open Atom with one or more directories, you will automatically get a Tree View on the side of your window.
+
+![Tree View in an open project](../../images/project-view.png "Tree View in an open project")
+
+The Tree View allows you to explore and modify the file and directory structure of your project. You can open, rename, delete and create new files from this view.
+
+You can also hide and show it with <kbd class="platform-mac">Cmd+\\</kbd><kbd class="platform-windows platform-linux">Ctrl+\\</kbd> or the `tree-view:toggle` command from the Command Palette, and <kbd class="platform-mac">Ctrl+0</kbd><kbd class="platform-windows platform-linux">Alt+\\</kbd> will focus it. When the Tree view has focus you can press <kbd class="platform-all">A</kbd>, <kbd class="platform-all">M</kbd>, or <kbd class="platform-all">Delete</kbd> to add, move or delete files and folders. You can also right-click on a file or folder in the Tree view to see many of the various options, including all of these plus showing the file in <span class="platform-mac">Finder</span><span class="platform-windows">Windows Explorer</span><span class="platform-linux">your native filesystem</span> or copying the file path to the clipboard.
+
+{{#note}}
+
+**Atom Packages**
+
+Like many parts of Atom, the Tree View is not built directly into the editor, but is its own standalone package that is shipped with Atom by default. Packages that are bundled with Atom are referred to as Core packages. Ones that aren't bundled with Atom are referred to as Community packages.
+
+You can find the source code to the Tree View on GitHub at https://github.com/atom/tree-view.
+
+This is one of the interesting things about Atom. Many of its core features are actually just packages implemented the same way you would implement any other functionality. This means that if you don't like the Tree View for example, you could write your own implementation of that functionality and replace it entirely.
+
+{{/note}}
+
+##### Opening a File in a Project
+
+Once you have a project open in Atom, you can easily find and open any file within that project.
+
+If you press <kbd class="platform-mac">Cmd+T</kbd><kbd class="platform-windows platform-linux">Ctrl+T</kbd> or <kbd class="platform-mac">Cmd+P</kbd><kbd class="platform-windows platform-linux">Ctrl+P</kbd>, the Fuzzy Finder will pop up. This will let you quickly search for any file in your project by typing parts of the path.
+
+![Opening files with the Fuzzy Finder](../../images/finder.png "Opening files with the Fuzzy Finder")
```

Comment:
Using the same text for `title` and `alt` attributes should be avoided, they serve very different purposes. It's annoying for people who rely on screenreaders to have the same text twice. Alt-texts are a delicate matter that requires a lot of thought to get them right.